### PR TITLE
Added home.yaml and about.yaml

### PR DESCRIPTION
**Pull request by haseokwoo**
Added home.yaml and about.yaml translated in Korean.

### CODE REVIEW

File: Localization/ko-KR/home.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+--- 
+language: en-US
+text:
```
Comment (fangfufu) [inline Localization/ko-KR/home.yaml]: Your ``home.yaml`` is longer than the one in en-US, so they don't match up. Could you update your local branch to include your, please?

File: Localization/ko-KR/about.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (fangfufu) [inline Localization/ko-KR/about.yaml]: Even though this passes through a YAML parser, it doesn't quite follow the coding style we have for other YAML files. 

1. Could you stick to the style of the other YAML files, please? 
2. Could you keep the original comments in, please? They serve as metadata for non-Korean speakers to figure out what we are looking at. However feel free to add more comments.

**Review by fangfufu — CHANGES_REQUESTED** [review]
Could you kindly apply the changes I requested, please? I will approve once the changes are made. :)

File: Localization/ko-KR/home.yaml (hunk 1 above)
Comment (haseokwoo) [reply Localization/ko-KR/home.yaml]: Sure I will. I asked you through the Discord Translate channel that the en-US version of home.yaml does not include the whole texts compare to the actual webpage.

I will make sure to match the texts with en-US and make a request.

File: Localization/ko-KR/about.yaml (hunk 2 above)
Comment (haseokwoo) [reply Localization/ko-KR/about.yaml]: I see. I think it was my misunderstood. I thought I need to parse the file and commit it as parsed. I will make changes and request soon,

File: Localization/ko-KR/home.yaml (hunk 1 above)
Comment (fangfufu) [reply Localization/ko-KR/home.yaml]: Ah, I meant, could you upload your version of en-US yaml and your more complete version of ko-KR yaml, please? We want the more complete version of home.yaml for en-US and ko-KR. We will get all other languages to follow your home.yaml.

**Review by fangfufu — APPROVED** [review]
I will approve this for now. Because it has met the inclusion standard.

File: Localization/ko-KR/about.yaml (hunk 2 above)
Comment (fangfufu) [reply Localization/ko-KR/about.yaml]: Thanks, I will mark this as resolved. :)

**Comment by fangfufu** [comment]
@haseokwoo, I have merged this, but please could you upload 
 - your more complete version of en-US yaml 
 - your more complete version of ko-KR yaml

We want the more complete version of home.yaml. We will get all other languages to follow your home.yaml. Thanks for being so helpful. :)

**Comment by haseokwoo** [comment]
@fangfufu Sure I will. Please let me know if any further things need to be added or changed. I will upload those soon.

**Pull request merged**